plugins/inputs/procstat/procstat.go

```diff
@@ -391,6 +457,62 @@ func (p *Procstat) SimpleFindPids(f PIDFinder) ([]PID, map[string]string, error)
 // execCommand is so tests can mock out exec.Command usage.
 var execCommand = exec.Command
 
+type MainStatus struct {
+	Map map[string]string
+}
+
+func BytesToString(b []byte) string {
+	return unsafe.String(&b[0], len(b))
+}
+
+func (p *Procstat) supervisorPIDs() ([]string, map[string]MainStatus, error) {
+	var Services string
```

Comment:
Local variables should never be exported!

Reply:
I removed this variable and wonder if the current writing method is reasonable?